pkgs/applications/office/docear/default.nix

```diff
@@ -0,0 +1,48 @@
+{stdenv, fetchurl, runtimeShell
+, oraclejre
+, antialiasFont ? true
+}:
+
+stdenv.mkDerivation rec {
+  pname = "docear";
+  version = "1.2";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "http://docear.org/downloads/docear_linux.tar.gz";
+    sha256 = "1g5n7r2x4gas6dl2fbyh7v9yxdcb6bzml8n3ldmpzv1rncgjcdp4";
+  };
+
+  phases = [ "unpackPhase" "installPhase" ];
```

Comment:
You should omit this as I believe it is or will be deprecated. Either way it isn't a good idea.